README.md

```diff
@@ -92,7 +92,7 @@ The frequency in primary or relapse tumors, for each unique variant and gene, is
 Note that the frequencies and counts may not tally as expected for several reasons. 
 First, the total columns use unique patients, while the primary/relapse tumor columns use unique samples. 
 Second, some submitters did not include information about the primary/relapse status of the samples, so those samples are omitted from the primary/relapse counts. 
-**Third, panel sample data is not included in the all cohorts calculation.** 
+Third, panel sample data is not included in the all cohorts calculation.
 Last, some patients or samples are included in multiple cohorts and may be counted multiple times. 
 See [SNV frequencies documentation](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/snv-frequencies) for details of how the unique variant ID, variant frequencies, and annotations were done using custom R scripts and see [see the hotspot detection documentation](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/hotspots-detection) for how SNV hotspots were called. 
```

Comment:
This one links to the AlexsLemonade/OpenPBTA git repo, while other links point to POT/OpenPedCan.

Reply:
Good catch, thank you!